app/graphql/mutations/save_account_mutation.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Mutations
+  # GraphQL mutation to update the current user account
+  class SaveAccountMutation
+    def call(user, args, _ctx)
```

Comment:
I think stripping all that away, we're left with lines 45 and 48 (and possibly the call to `set_minimum_password_length`, whatever that does), which is the same the mutation currently does.